WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift

```diff
@@ -97,16 +97,27 @@ private extension PluginListViewController {
 // MARK: - Actions
 //
 private extension PluginListViewController {
+
+    /// resyncPlugins is used when the view need to resynchronize data
+    ///
     @objc func resyncPlugins() {
+        syncPlugins(onCompletion: nil)
+    }
+
+    /// syncPlugins synchronizes all plugins and execute onCompletion when is finished
+    ///
+    @objc func syncPlugins(onCompletion: (() -> Void)?) {
         removeErrorStateView()
         startGhostAnimation()
-        viewModel.resyncPlugins { [weak self] result in
+        viewModel.syncPlugins { [weak self] result in
             guard let self = self else { return }
             self.refreshControl.endRefreshing()
             self.stopGhostAnimation()
             if result.isFailure {
                 self.displayErrorStateView()
             }
+            guard let onCompletion = onCompletion else { return }
+            onCompletion()
```

Comment:
I think this is simpler and more common in our code base :-)
```suggestion
            onCompletion?()
```

Reply:
Hi! I was using the guard to avoid the use of the optional. I will change it.